packages/plugin-ext/src/hosted/node/plugin-host-rpc.ts

```diff
@@ -14,13 +14,14 @@
  * SPDX-License-Identifier: EPL-2.0 OR GPL-2.0 WITH Classpath-exception-2.0
  ********************************************************************************/
 
+import { dynamicRequire } from '@theia/core/lib/node/dynamic-require';
```

Comment:
@eclipse-theia/plugin-system does that `@theia/core` import cause any issue with isolating the host process or anything else I would be missing?

Reply:
I don't think so: as long as the module is available on the back end, we should be fine.